### ISSUE TITLE
Duplicate CONCEPT_CODES in VOCABULAR_ID = 'DPD'

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#89]
This is for 20160817.

There is some duplication going on in the DPD Vocabulary.

``` sql
SELECT *
FROM CONCEPT
WHERE CONCEPT_CODE IN (
'779830','449989','651613','517224','517232','Not Applicable/non applicable','354279','732052','872695','532266',
'578746','493562','346632','479500','274577','306584','430862','139696','336858','517208',
'488372','564826','2308762','328804','274860','545384','470910'
)
AND VOCABULARY_ID = 'DPD'
ORDER BY VOCABULARY_ID, CONCEPT_CODE
```

For example, the CONCEPT_CODE `274577` in the VOCABULARY_ID `DPD` comes up with the following CONCEPTs:

``` text
44688390    WORMER ONCE A MONTH FOR CATS KITTENS CAP 84MG
44687290    WORMER ONCE A MONTH FOR PUPPIES CAP 84MG
44689071    WORMER ONCE A MONTH FOR DOGS CAP 84MG
```

**Comment by ericaVoss** [comment]
Duplicate of #83

**Comment by ericaVoss** [comment]
This problem is fixed in 20161107.

Tied to this open issue:
https://github.com/OHDSI/Vocabulary-v5.0/issues/95

However these missing concepts may be due to fixing this duplicate issue.
